include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -1506,6 +1506,15 @@ FUNCTION(TaskCreateFunc,
          ARGS(SizeTy, SwiftTaskPtrTy, AsyncFunctionPointerPtrTy),
          ATTRS(NoUnwind, ArgMemOnly))
 
+// void swift_asyncSuspend(
+//     Executor *current, Executor *target, AsyncTask *task);
+FUNCTION(AsyncSuspendFunc,
+         swift_asyncSuspend, SwiftCC,
```

Comment:
Can we call this `swift_task_suspend` or similar, because all of the task stuff starts with `swift_task_`.